executor-service/src/main/java/executor/service/config/SecurityConfig.java

```diff
@@ -18,15 +20,16 @@
 @EnableWebSecurity
 @PropertySource("classpath:application.properties")
 public class SecurityConfig {
-
+    @Autowired
+    private RsaManager rsaManager;
```

Comment:
add `final` to prevent NPE and to autowire the dependency:
```suggestion
    private final RsaManager rsaManager;
```